### ISSUE TITLE
Immich App for Android fails to play certain videos and crash

### ISSUE THREAD

**Issue by chowhi123** [immich-app/immich#7539]
### The bug

Immich App for Android fails to play certain videos and crash

### The OS that Immich Server is running on

Ubuntu 22.04 LTS

### Version of Immich Server

v1.97.0

### Version of Immich Mobile App

v1.97.0 build 125

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
version: "3.8"

#
# WARNING: Make sure to use the docker-compose.yml of the current release:
#
# https://github.com/immich-app/immich/releases/latest/download/docker-compose.yml
#
# The compose file on main may not be compatible with the latest release.
#

name: immich

services:
  immich-server:
    container_name: immich_server
    image: ghcr.io/immich-app/immich-server:$IMMICH_VERSION
    command: [ "start.sh", "immich" ]
    volumes:
      - $UPLOAD_LOCATION:/usr/src/app/upload
      - /etc/localtime:/etc/localtime:ro
    environment:
      - DB_USERNAME=$DB_USERNAME
      - DB_PASSWORD=$DB_PASSWORD
      - DB_DATABASE_NAME=$DB_DATABASE_NAME
      - DB_HOSTNAME=$DB_HOSTNAME
    ports:
      - 8010:3001
    depends_on:
      - redis
      - database
    restart: always

  immich-microservices:
    container_name: immich_microservices
    image: ghcr.io/immich-app/immich-server:$IMMICH_VERSION
    # extends: # uncomment this section for hardware acceleration - see https://immich.app/docs/features/hardware-transcoding
    #   file: hwaccel.transcoding.yml 
    #   service: cpu # set to one of [nvenc, quicksync, rkmpp, vaapi, vaapi-wsl] for accelerated transcoding
    command: [ "start.sh", "microservices" ]
    volumes:
      - $UPLOAD_LOCATION:/usr/src/app/upload
      - /etc/localtime:/etc/localtime:ro
    environment:
      - DB_USERNAME=$DB_USERNAME
      - DB_PASSWORD=$DB_PASSWORD
      - DB_DATABASE_NAME=$DB_DATABASE_NAME
      - DB_HOSTNAME=$DB_HOSTNAME
    depends_on:
      - redis
      - database
    restart: always

  immich-machine-learning:
    container_name: immich_machine_learning
    # For hardware acceleration, add one of -[armnn, cuda, openvino] to the image tag.
    # Example tag: ${IMMICH_VERSION:-release}-cuda
    image: ghcr.io/immich-app/immich-machine-learning:$IMMICH_VERSION
    # extends: # uncomment this section for hardware acceleration - see https://immich.app/docs/features/ml-hardware-acceleration
    #   file: hwaccel.ml.yml
    #   service: cpu # set to one of [armnn, cuda, openvino, openvino-wsl] for accelerated inference - use the `-wsl` version for WSL2 where applicable
    volumes:
      - model-cache:/cache
    restart: always

  redis:
    container_name: immich_redis
    image: redis:6.2-alpine@sha256:51d6c56749a4243096327e3fb964a48ed92254357108449cb6e23999c37773c5
    restart: always

  database:
    container_name: immich_postgres
    image: tensorchord/pgvecto-rs:pg14-v0.2.0@sha256:90724186f0a3517cf6914295b5ab410db9ce23190a2d9d0b9dd6463e3fa298f0
    environment:
      - POSTGRES_PASSWORD=$DB_PASSWORD
      - POSTGRES_USER=$DB_USERNAME
      - POSTGRES_DB=$DB_DATABASE_NAME
    volumes:
      - /data/immich/pgdata:/var/lib/postgresql/data
    restart: always

volumes:
  pgdata:
  model-cache:
```

### Your .env content

```Shell
# You can find documentation for all the supported env variables at https://immich.app/docs/install/environment-variables

# The location where your uploaded files are stored
UPLOAD_LOCATION=/mnt/immich

# The Immich version to use. You can pin this to a specific version like "v1.71.0"
IMMICH_VERSION=release

# Connection secret for postgres. You should change it to a random password
DB_PASSWORD=postgres

# The values below this line do not need to be changed
###################################################################################
DB_HOSTNAME=immich_postgres
DB_USERNAME=postgres
DB_DATABASE_NAME=immich

REDIS_HOSTNAME=immich_redis
```

### Reproduction steps

```bash
1.Open the immich app
2.Play Video
3.App would crash
...
```

### Additional information
Using Nginx Proxy Manager, Cloudflare domain, SSL with lets encrypt and CF DNS challenge

Galaxy S23+ (SM-S916U)
One UI 6.0
Android 14
![KakaoTalk_20240229_140142655](https://github.com/immich-app/immich/assets/61570417/f980c94f-9316-43f5-b3c5-c700615dab32)

**Comment by chowhi123** [comment]
Issue fixed after reinstalling the app

**Comment by chowhi123** [comment]
Tested with my wife's iPhone 15 using partner function, it seems to work fine on her end

**Comment by alextran1502** [comment]
Does the video come from your account or your partner's?

**Comment by chowhi123** [comment]
> 

The video comes from mine.

**Comment by chowhi123** [comment]
I reinstalled which didn't help with the issue. I downloaded 1.96.0 build.124 works fine. There definitely is an issue with 1.97 android app. Thank you for all developers!

**Comment by nebb00** [comment]
I have the same issue on 1.97 as android. Play video, immediately crashes

**Comment by chowhi123** [comment]
@martyfuhry I saw [#7486](https://github.com/immich-app/immich/pull/7486) which is probably the only change in 1.97 mobile that might have caused this issue. Could you take a look at it for me please?

**Comment by martyfuhry** [comment]
Sorry, that was probably me! We noticed a few other issues around video playback like #7562 since I went in monkeying with the video player.

I just got done with a big refactor #7625 which may address this. We are also curious about switching to a new video player which may be more stable and have better support.

**Comment by chowhi123** [comment]
> Sorry, that was probably me! We noticed a few other issues around video playback like #7562 since I went in monkeying with the video player.
> 
> I just got done with a big refactor #7625 which may address this. We are also curious about switching to a new video player which may be more stable and have better support.

Thank you for taking a look!

**Comment by chowhi123** [comment]
This has been resolved on 1.98
